services/src/main/java/org/broadleafcommerce/payment/service/gateway/PayPalCheckoutRestConfiguration.java

```diff
@@ -0,0 +1,173 @@
+package org.broadleafcommerce.payment.service.gateway;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadleafcommerce.vendor.paypal.service.PayPalWebProfileService;
+import org.broadleafcommerce.vendor.paypal.service.payment.MessageConstants;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import com.broadleafcommerce.paymentgateway.domain.PaymentRequest;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+/**
+ * @author Elbert Bautista (elbertbautista)
+ * @author Chris Kittrell
+ */
+@ConfigurationProperties("gateway.paypal.checkout.rest")
```

Comment:
Since you're changing this already, change the property names to start with `broadleaf`, e.g. `braodleaf.paypal.checkout.rest`